src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -660,15 +660,29 @@ export default class Pilot {
      *
      * @for pilot
      * @method conductInstrumentApproach
-     * @param approachType {string}       the type of instrument approach (eg 'ILS', 'RNAV', 'VOR', etc)
-     * @param runwayModel {RunwayModel}   the runway the approach ends at
-     * @return {array}                    [success of operation, readback]
+     * @param aircraftModel {AircraftModel} the aircraft model belonging to this pilot
```

Comment:
does an aircraft model belong to a pilot?  or does the pilot belong to an aircraft model?